scripts/discourse-sync/client.py

```diff
@@ -0,0 +1,22 @@
+import pydiscourse
+
+# Client extends pydiscourse.DiscourseClient with new methods
+class Client(pydiscourse.DiscourseClient):
+    def lock_post(self, post_id, locked, **kwargs):
```

Comment:
This is a nit, but this looks very Go :smile:. Python you'd typically do something like:

```suggestion
class Client(pydiscourse.DiscourseClient):
    """Extends pydiscourse.DiscourseClient to support locking and staff notices."""

        def lock_post(self, post_id, locked, **kwargs):
```

Reply:
I am a Go programmer after all :D